build.gradle

```diff
@@ -108,6 +108,7 @@ configure(projectsToConfigure) { projectToConf ->
 
   projectToConf.ext.intellijHome = projectToConf.hasProperty('intellijHome') ? intellijHome : System.getenv('INTELLIJ_HOME')
   projectToConf.ext.intellijSandboxDir = System.getenv('SAROS_INTELLIJ_SANDBOX')
+  projectToConf.ext.versionQualifier = projectToConf.hasProperty('versionQualifier') ? '.' + versionQualifier : ''
```

Comment:
Does the qualifier count as "set" if it is defined as empty? I.e. does this change mean that the versions will always end in a '.' as the property `versionQualifier` is set by default (even though it is set to be empty).

*Update:* This is in fact the case. You can't start a local instance using the `runIde` task using this branch as it fails due to an illegal plugin version modifier `0.2.2.`, meaning the trailing `.` is included when no version qualifier is set.

Reply:
Removing the default entry in `gradle.properties` resolves this issue.

Reply:
We could consider using "DEV" as a default to make it easier to determine whether we are working on a dev build or not (and make it easier to tell if user report issues on their own build).

The issue with that is that I don't know whether you can "un-set" a parameter. We would still need a way of building artifacts with the normal version for release. So that should be a possibility without having to modify the source (i.e. removing the `gradle.properties` entry).

Reply:
Thank you :+1:

> The issue with that is that I don't know whether you can "un-set" a parameter.

You can call `./gradlew -PversionQualifier= sarosEclipse` looks weird but results in an empty property value.